packages/protocol/test/signal/SignalService.integration.test.ts

```diff
@@ -0,0 +1,97 @@
+import { expect } from "chai";
+import { ethers } from "hardhat";
+import { TestHeaderSync } from "../../typechain";
+import { deploySignalService, getSignalProof } from "../utils/signal";
+import deployAddressManager from "../utils/addressManager";
+import { getBlockHeader } from "../utils/rpc";
+
+describe("integration:SignalService", function () {
+    async function deployIntegrationSignalServiceFixture() {
+        const [owner, nonOwner] = await ethers.getSigners();
+
+        const { chainId } = await ethers.provider.getNetwork();
```

Comment:
Will this work:

`const { chainId: srcChainId} = await ethers.provider.getNetwork();`

Reply:
I'll try it.